### ISSUE TITLE
Add alt text fields

### ISSUE THREAD

**Issue by nmorduch** [newamericafoundation/newamerica-cms#1775]
See https://docs.wagtail.org/en/stable/advanced_topics/accessibility_considerations.html#alt-text-for-images

Story images also need alt text, and we should check for other places that have images

**Comment by nmorduch** [comment]
Image block should have alt text and caption
Story image should just have alt text, no caption
Can you send me a list of other places there are images and I'll assess what they need?
